controllers/searchoperator_controller.go

```diff
@@ -146,6 +148,10 @@ func (r *SearchOperatorReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 
 	// Setup RedisGraph Deployment
 	r.Log.Info(fmt.Sprintf("Config in  Use Persistence/AllowDegrade %t/%t", persistence, allowdegrade))
+	if !setupPod {
+		r.Log.Info("Not deploying redisgraph. Using custom/existing deployment of Redisgraph pod")
```

Comment:
```suggestion
		r.Log.Warn("Not deploying the database. This is not an error, it's a current limitation in this environment. The search feature is not operational.  More info: https://github.com/open-cluster-management/community/issues/34")
```

Reply:
Logger doesn't support `Warnings`. So, converted to `Info`.

Reply:
ok, didn't realize that.